benchmarks/python/test_layernorm_fwd.py

```diff
@@ -64,7 +83,7 @@ def test_layernorm_fwd_benchmark(
     clear_cuda_cache()
 
```

Comment:
Unwrapping the `size` tuple might help with clarity here since `size[1]` is a little tough to decipher at a glance.